android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt

```diff
@@ -1,13 +1,479 @@
+			TextField(
+				modifier = Modifier
+					.fillMaxWidth(ELEMENT_WIDTH),
+				singleLine = true,
+				shape = RoundedCornerShape(size = 100.dp),
+				colors = TextFieldDefaults.colors(
+					unfocusedContainerColor = buttonBackgroundColor,
+					focusedContainerColor = buttonBackgroundColor,
+					unfocusedIndicatorColor = Color.Transparent,
+					focusedIndicatorColor = Color.Transparent,
+				),
+				visualTransformation = PasswordVisualTransformation(),
+				keyboardOptions = KeyboardOptions(
+					keyboardType = KeyboardType.Password,
+					autoCorrect = false
+				),
+				placeholder = {
+					Text(
+						text = stringResource(id = R.string.confirm_password),
+						color = secondaryTextColor,
+					)
+				},
+				value = uiState.confirmPassword,
+				onValueChange = {
+					viewModel.updateUiState(
+						CreateAccountPageUiState.Loaded(
+							firstName = uiState.firstName,
+							lastName = uiState.lastName,
+							email = uiState.email,
+							password = uiState.password,
+							confirmPassword = it,
+							gender = uiState.gender,
+						)
+					)
+				}
+			)
+
+			Spacer(
+				modifier = Modifier.weight(weight = 2.0f)
+			)
+
+			Box {
+				Button(
+					modifier = Modifier
+						.fillMaxWidth(ELEMENT_WIDTH)
+						.height(50.dp),
+					onClick = {
+						expandedDropdown = true
+					},
+					colors = ButtonDefaults.buttonColors(
+						containerColor = buttonBackgroundColor,
+						contentColor = secondaryTextColor,
+					)
+				) {
+					Row(
+						verticalAlignment = Alignment.CenterVertically
+					) {
+						Text(
+							text = stringResource(id = uiState.gender.stringId),
+						)
+
+						Spacer(modifier = Modifier.weight(1.0f))
+
+						Icon(
+							imageVector = Icons.Filled.KeyboardArrowDown,
+							contentDescription = "genders"
+						)
+					}
+				}
+				DropdownMenu(
+					expanded = expandedDropdown,
+					onDismissRequest = { expandedDropdown = false },
+					modifier = Modifier
+						.fillMaxWidth(ELEMENT_WIDTH)
+				) {
+					enumValues<Gender>().forEach { choice ->
+						DropdownMenuItem(
+							text = {
+								Text(text = stringResource(id = choice.stringId))
+							},
+							onClick = {
+								viewModel.updateUiState(
+									CreateAccountPageUiState.Loaded(
+										firstName = uiState.firstName,
+										lastName = uiState.lastName,
+										email = uiState.email,
+										password = uiState.password,
+										confirmPassword = uiState.confirmPassword,
+										gender = choice,
+									)
+								)
+								expandedDropdown = false
+							}
+						)
+					}
+				}
+			}
+
+			Spacer(
+				modifier = Modifier.weight(weight = 2.0f)
+			)
+
+			Button(
+				modifier = Modifier
+					.fillMaxWidth(ELEMENT_WIDTH)
+					.height(50.dp),
+				onClick = {},
+				colors = ButtonDefaults.buttonColors(
+					containerColor = subletrPink,
+					contentColor = Color.White,
+				)
+			) {
+				Text(
+					text = stringResource(id = R.string.create_account),
+					color = Color.White,
```

Comment:
Can you use the defined constant for this?
I think it's called `textOnSubletrPink` or something like that.
It'll just make our lives easier if we want to implement dark mode

Reply:
I just added that constant for white text color and am using it now

Reply:
disregard that i need to resolve conflicts lol